reporters/GlobalSummaryReporter.js

```diff
@@ -187,21 +194,33 @@ class GlobalSummaryReporter {
     this._generateSponsorStats(periods);
     console.groupEnd();
 
-    // 2. Tokens stats table
+    // 2a. Tokens Swaps table
+    if (this.latestSwapTimestamp) {
```

Comment:
> Maybe I'm misunderstanding, but is this just a way to test for whether we have valid exchange data to display?

Correct. I added this `bool` to the end of the `update()` call:
```
    // Have we successfully fetched any exchange data from the graph API?
    this.hasExchangeData = this.latestSwapTimestamp;
```

It will be true if we are able to get any data from an exchange subgraph